README.md

```diff
@@ -79,3 +79,14 @@ Optionally, one may also run the `check_audio_files.pl` script (in the
 `scripts` directory of the [main](https://github.com/De7vID/klingon-assistant)
 repo) to see if any syllables have been added which are not available in the
 [TTS](https://github.com/De7vID/klingon-assistant-tts-android).
+
+Conventions for German translators
```

Comment:
You can preview your edit by selecting "View file" in the menu on the right. It's a good idea to do so before making a pull request.

The file is in markdown (.md) format, which means that things like headings are marked. Otherwise (as you can see in the preview), this is treated as a regular line (attached to the next line).

I'd suggest to make a top-level heading like "Conventions for translators" and a subheading "German", but I can do that later. (I actually want to add conventions for other languages also.)

Reply:
Done. What's the next step to add this to the boQwI'?